epyqlib/devicetreeview.py

```diff
@@ -221,6 +230,28 @@ def add_device(self, bus, device=None):
 
         return device
 
+    def change_device_number(self, node):
+        pcan_bus = can.interface.Bus(bustype=node.interface, channel=node.channel)
+        current_device_number = pcan_bus.get_device_number()
+        new_device_number, ok = QInputDialog.getInt(
+            None,
+            *(("Device ID",) * 2),
```

Comment:
You enjoy some occasional golf, I see.  `:]`

Reply:
Speaking of the "Converter Node ID" input dialog, that's where I stole this definition from. So you see, I learned to golf from a golf pro.